vispTypes.hpp

```diff
@@ -38,6 +39,23 @@ namespace visp
         float gain_at_infinity;
         float slope_at_zero;
     };
+
+    /** Group relevant information about the controller
+     *
+     *  error        - the norm-2 of the error vector
+     *  residual     - value expressed in meter that represents the
+                       "quality" of the measurement"
+     *  desired_pose - the current setpoint
+     *  current_pose - the current pose
+     */
+    struct controllerState
+    {
+        double error;
```

Comment:
I've added. 
However, I was checking that in [here](https://github.com/rock-control/control-orogen-visp/blob/aa6c35c9869a857937fd1939768f55c72e12c72e/tasks/Task.cpp#L291) has already a time of the data. Do you think that is important to inform it explicitly?